### DISCUSSION TITLE
TuYa TS1101_dimmer_module    minimum Brightness not working

### DISCUSSION THREAD

**Discussion by RafaelDuncan** [Koenkk/zigbee2mqtt#9734] [Devices]
Today I found out that a new supported device has recently been added called "TuYa TS1101_dimmer_module.
I have this device working now and am happy with it.

What I just don't understand is the following:
I have entered a value at min_brightness, but when I dim the dimmer below this value, this is simply possible. I can therefore dim it to value "0" and that is not the intention. does anyone have an idea how it is possible that it does nothing with the set min_brightness value?
"

**Comment by RafaelDuncan** [comment]
I have now restarted Zigbee module and now it works fine.

**Reply by upandabout** [reply]
How did you restart the zigbee module? Are you talking about hardware or software here??

**Reply by RafaelDuncan** [reply]
I had to think again since this was a few years ago. I think removing power from the built-in module was the solution. If that doesn't work, try pairing it again.

**Reply by upandabout** [reply]
OK thanks.
But where do you really enter the minimum level? I was trying it from the dev console, and through that mannaged to set a minimum level, but now I'm unable to switch the dimmer off? Edit, so not being able to tun off this dimmer, this was due to me trying to input a transition time, which this particulier Tuya dimmer TS0052 doesn't work. If you end in that situation, delete the value in the transition field and it should work again.

Kind regards Kasper
